app/components/candidate_interface/application_dashboard_course_choices_component.html.erb

```diff
@@ -7,18 +7,8 @@
     )) do %>
       <div class="app-summary-card__actions">
         <%= govuk_link_to 'View application', candidate_interface_application_review_submitted_path %>
-        <% if !application_choice.offer? && !withdrawable?(application_choice) && @editable %>
-          <%= govuk_link_to candidate_interface_confirm_destroy_course_choice_path(application_choice.id), data: { action: :delete } do %>
-            <%= t('application_form.courses.delete') %>
-            <span class="govuk-visually-hidden"> <%= application_choice.current_course.name_and_code %></span>
-          <% end %>
-        <% end %>
```

Comment:
This is only possible for unsubmitted 2023 applications and because is not possible to add/edit/remove course choices from 2023 we can safely delete it.